### ISSUE TITLE
Telegram CLI

### ISSUE THREAD

**Issue by zmoog** [zmoog/telegram-cli#1]
I want a simple tool to send messages to a Telegram user or group:

```shell
$ telegram send --message "Hello world!"
```

or

```shell
$ telegram send --message-file message.txt
```

**Comment by zmoog** [comment]
I created Telegram API client before, let's search for my previous work.

**Comment by zmoog** [comment]
Okay, sending messages is not complex! :-) 

From https://github.com/zmoog/hogwarts/blob/master/app/notification/telegram.py here is the source code:

```python
import requests


class TelegramNotifier:

    def __init__(self, token, chat_id):
        self.token = token
        self.chat_id = chat_id

    def send(self, msg):
        params = {
            'chat_id': self.chat_id,
            'text': msg
        }
        r = requests.get(
            f'https://api.telegram.org/{self.token}/sendMessage',
            params=params
        )
```

**Comment by zmoog** [comment]
I am making some adjustments:

```python
import os

import requests


class Client:
    """Telegram client."""

    def __init__(self, token: str):
        self.token = token

    @staticmethod
    def from_envorinment() -> "Client":
        """Create a client from the TELEGRAM_TOKEN environment variable."""

        if "TELEGRAM_TOKEN" not in os.environ:
            raise Exception(
                "TELEGRAM_TOKEN environment variable not found. "
                "Please set it to your Telegram API token."
            )

        return Client(token=os.environ['TELEGRAM_TOKEN'])

    def send(self, msg: str, chat_id: str):
        """Send a message to a chat."""

        params = {
            'chat_id': chat_id,
            'text': msg
        }

        r = requests.get(
            f'https://api.telegram.org/{self.token}/sendMessage',
            params=params
        )

        r.raise_for_status()
```

Now I need a chat_it to send messages to.

**Comment by zmoog** [comment]
According to https://stackoverflow.com/questions/32423837/telegram-bot-how-to-get-a-group-chat-id, here's the quickest way to get a group chat id:

```shell
$ curl -i -XGET "https://api.telegram.org/TOKEN/getUpdates"
HTTP/2 200 
server: nginx/1.18.0
date: Thu, 09 Feb 2023 05:18:02 GMT
content-type: application/json
content-length: 336
strict-transport-security: max-age=31536000; includeSubDomains; preload
access-control-allow-origin: *
access-control-allow-methods: GET, POST, OPTIONS
access-control-expose-headers: Content-Length,Content-Type,Date,Server,Connection
{
  "ok": true,
  "result": [
    {
      "update_id": 123,
      "message": {
        "message_id": 123,
        "from": {
          "id": 123,
          "is_bot": false,
          "first_name": "Maurizio",
          "last_name": "Branca",
          "username": "zmoog",
          "language_code": "en"
        },
        "chat": {
          "id": 161035319,
          "first_name": "Maurizio",
          "last_name": "Branca",
          "username": "zmoog",
          "type": "private"
        },
        "date": 1675919879,
        "text": "hey"
      }
    }
  ]
}
```

**Comment by zmoog** [comment]
Final version:

    $ tgm message send --chat-id 123456789 --text 'Hello'
    message-id: 676
    
Closed by #2
